src/stacked_borrows.rs

```diff
@@ -467,6 +469,51 @@ impl<'tcx> Stack {
 
         Ok(())
     }
+
+    /// Report a descriptive error when `new` could not be granted from `derived_from`.
+    fn grant_error(
+        &self,
+        derived_from: SbTag,
+        new: Item,
+        alloc_id: AllocId,
+        alloc_range: AllocRange,
+    ) -> InterpError<'static> {
+        let action = format!(
+            "trying to reborrow for {:?} over {:?}[{:#x}..{:#x}]",
```

Comment:
"over" a memory range sounds like an odd choice of proposition to me? I am not a native speaker, but I would have expected "in" or "at".

Reply:
Preposition choice is one of the most arcane parts of English. I don't think there are any choices I'm confident will read well to all users, so I'm trying to refactor it out.